src/core/xccl_context.c

```diff
@@ -28,17 +29,17 @@ static xccl_status_t find_tlib_by_name(xccl_lib_t *lib, const char *tlib_name,
 xccl_status_t xccl_context_create(xccl_lib_h lib,
                                   const xccl_context_params_t *params,
                                   const xccl_context_config_t *config,
-                                  xccl_context_h *context)
-{
-    xccl_context_t    *ctx            = malloc(sizeof(xccl_context_t));
```

Comment:
@Sergei-Lebedev @lappazos we have a lot of "indent" changes here, do we need them? Before the change assignments have been aligned on "=", now they are not. Why?

Reply:
fixed